Portfolio.html

```diff
@@ -3,12 +3,86 @@
 <head>
     <meta charset="UTF-8">
     <link href="https://fonts.googleapis.com/css?family=Quicksand" rel="stylesheet">
-    <link rel="stylesheet" type="text/css" href="Portfolio.css">
-    <title>Rob Reeves Portfolio</title>
+    <link href="https://fonts.googleapis.com/css?family=Fanwood+Text" rel="stylesheet">
+    <link rel="stylesheet" type="text/css" href=":css/normalize.css">
```

Comment:
if you remove the `/` from your folder names, you wont need `:` in your links

Reply:
fixed.